x-pack/legacy/plugins/upgrade_assistant/public/np_ready/application/components/tabs.tsx

```diff
@@ -157,7 +153,9 @@ export class UpgradeAssistantTabsUI extends React.Component<
   private loadData = async () => {
     try {
       this.setState({ loadingState: LoadingState.Loading });
-      const resp = await axios.get(chrome.addBasePath('/api/upgrade_assistant/status'));
+      const resp = await axios.get(
```

Comment:
Here and in other places: using the axios client won't give you the benefits of the `core.http.fetch` client like automatically adding the basepath, xsrf token and in the future redirecting a user to the login page when the kibana server returns 401 requests. So I would recommend switching over at some point.

Reply:
I will make a note of this!